AuthoringToolTest/PresentationLogic/ModalDialog/ModalDialogInputFieldsFactoryUt.cs

```diff
@@ -9,6 +15,277 @@ namespace AuthoringToolTest.PresentationLogic.ModalDialog;
 
 public class ModalDialogInputFieldsFactoryUt
 {
+    [Test]
+    public void GetCreateLearningElementInputFields_Test()
+    {
+        var systemUnderTest = new ModalDialogInputFieldsFactory();
```

Comment:
Bitte systemUnderTest erst erstellen, nachdem alles andere vorbereitet wurde. Das Pattern nennt sich Arrange-Act-Assert (https://docs.telerik.com/devtools/justmock/basic-usage/arrange-act-assert)

Zudem bitte das `new` in eine `GetModalInputFieldsFactoryForTesting` Methode auslagern, ähnlich wie wir das in anderen Tests gemacht haben.

Reply:
also so?
`var systemUnderTest = GetModalInputFieldsFactoryForTesting();`